code/turf/floors_airless.dm

```diff
@@ -113,14 +113,23 @@
 /turf/unsimulated/floor/airless/plating/catwalk/auto/iomoon
 	name = "hot catwalk support"
 	icon_state = "auto_lava"
-	carbon_dioxide = 20
-	temperature = FIRE_MINIMUM_TEMPERATURE_TO_EXIST - 1
+	var/is_safe = FALSE
 
 	New()
+		if (!src.is_safe)
+			src.carbon_dioxide = 20
+			src.temperature = FIRE_MINIMUM_TEMPERATURE_TO_EXIST - 1
+		else // put under unsimulated to not repeat code
+			src.oxygen = MOLES_O2STANDARD
+			src.nitrogen = MOLES_N2STANDARD
+			src.temperature = T20C
 		..()
 		var/image/lava = image(icon = 'icons/turf/floors.dmi', icon_state = "lava", layer = src.layer - 0.1)
 		src.UpdateOverlays(lava, "lava")
 
+/turf/unsimulated/floor/airless/plating/catwalk/auto/iomoon/safe
+	is_safe = TRUE
+
```

Comment:
Why not just have the `.../safe` type override the vars on the type, rather than having them be set in `New()`? I don't see what the point of this pattern is.

Reply:
This was left in accidentally, I didn't mean to make a subtype - will be no changes to this file